CHANGELOG.md

```diff
@@ -14,7 +14,7 @@ and `a-btn__icon-on-right` classes referenced in docs to correct
 `a-btn_icon__on-left` and `a-btn_icon__on-right` classes.
 
 ### Removed
--
+- **capital-framework:** [MINOR] Remove IE7 and below support.
```

Comment:
Broken into individual modules in https://github.com/cfpb/capital-framework/pull/514/commits/01403fa2de19eaa100e0e6f7dcaf62864e7679ab